TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO;
+namespace TeleSharp.TL.Channels
+{
+    [TLObject(870184064)]
+    public class TLRequestGetAdminLog : TLMethod
+    {
+        public override int Constructor
+        {
+            get
+            {
+                return 870184064;
+            }
+        }
+
+        public int flags { get; set; }
```

Comment:
@aarani do you mind fixing this to be PascalCase? It's not respecting the MS .NET API guidelines.

Reply:
Pascal or Camel Case ?

Reply:
camelCase is for Java (lowercase the first letter, uppercase each new word), but for .NET it is PascalCase (uppercase the first letter of each word, even the first one)

Reply:
so it should be "Flags" instead of "flags", and "EventsFilter" instead of "events_filter".

Reply:
(for method names, properties and public fields; private fields can be left as camelCase)

Reply:
#627